src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/tasks/ReprocessExistingMovesTask.kt

```diff
@@ -16,16 +18,38 @@ import uk.gov.justice.digital.hmpps.pecs.jpc.util.DateRange
  * Even though it will be going through all the historic moves it will do it on a month by month basis to prevent
  * processing large quantities of data.
  */
+
+private val logger = loggerFor<ReprocessExistingMovesTask>()
+
 @Component
 class ReprocessExistingMovesTask(
-  private val service: HistoricMovesProcessingService,
+  private val processingService: HistoricMovesProcessingService,
+  private val moveService: MoveService,
   private val timeSource: TimeSource,
   monitoringService: MonitoringService
 ) : Task("Reprocess historic moves", monitoringService) {
 
   override fun performTask() {
-    DateRange(EffectiveYear.startOfContract(), timeSource.yesterday()).forEachMonth { month ->
-      Supplier.forEach { supplier -> service.process(month, supplier) }
+    val dateRange = DateRange(EffectiveYear.startOfContract(), timeSource.yesterday())
+
+    Supplier.forEach { supplier ->
+      logger.info("Reprocessing historic moves for $supplier")
+
+      logSupplierMoveSummaries(supplier, dateRange)
```

Comment:
Logging the numbers before and after so we can track in the logs the actual number of move types changes changes (if any).